src/pkg/logger/logger.go

```diff
@@ -54,34 +65,48 @@ func AddLogLevelFlag(cmd *cobra.Command) {
 	fs.MarkHidden(readableLogsFN)
 }
 
-// Due to races between the lazy evaluation of flags in cobra and the need to init logging
-// behavior in a ctx, log-level gets pre-processed manually here using pflags.  The canonical
-// AddLogLevelFlag() ensures the flag is displayed as part of the help/usage output.
-func PreloadLogLevel() string {
+// Due to races between the lazy evaluation of flags in cobra and the
+// need to init logging behavior in a ctx, log-level and log-file gets
+// pre-processed manually here using pflags.  The canonical
+// AddLogLevelFlag() and AddLogFileFlag() ensures the flags are
+// displayed as part of the help/usage output.
+func PreloadLoggingFlags() (string, string) {
 	fs := pflag.NewFlagSet("seed-logger", pflag.ContinueOnError)
 	fs.ParseErrorsWhitelist.UnknownFlags = true
 	fs.String(logLevelFN, "info", "set the log level to debug|info|warn|error")
+	fs.String(logFileFN, defaultLogLocation, "location for writing logs")
 	fs.BoolVar(&DebugAPI, debugAPIFN, false, "add non-2xx request/response errors to logging")
 	fs.BoolVar(&readableOutput, readableLogsFN, false, "minimizes log output: removes the file and date, colors the level")
 	// prevents overriding the corso/cobra help processor
 	fs.BoolP("help", "h", false, "")
 
 	// parse the os args list to find the log level flag
 	if err := fs.Parse(os.Args[1:]); err != nil {
-		return "info"
+		return "info", defaultLogLocation
 	}
 
 	// retrieve the user's preferred log level
 	// automatically defaults to "info"
 	levelString, err := fs.GetString(logLevelFN)
 	if err != nil {
-		return "info"
+		return "info", defaultLogLocation
+	}
+
+	// retrieve the user's preferred log file location
+	// automatically defaults to default log location
+	logfile, err := fs.GetString(logFileFN)
+	if err != nil {
+		return "info", defaultLogLocation
+	}
+
+	if logfile == "-" {
```

Comment:
They can still use cobra specific `stdout`, but this is a more common convention.